prog/hints.go

```diff
@@ -236,7 +276,8 @@ func shrinkExpand(v uint64, compMap CompMap, bitsize uint64) []uint64 {
 				if bigendian {
 					newV = swapInt(newV, width)
 				}
-				if specialIntsSet[newV] {
+				// Images are large so it's hard to guess even special ints with random mutations.
+				if !image && specialIntsSet[newV] {
```

Comment:
The original if condition denied the replacement of bytes to special integers, right?

And now you allowed such replacement, right?

I just don't really get the motivation neither before nor after the changes.

Reply:
I've updated the comment. PTAL.